_posts/2015-03-12-0.35-release.md

```diff
@@ -0,0 +1,10 @@
+---
+title:  "Release 0.35"
+author: Julien Pagès
+---
+
+This 0.35 release implement a new algorithm to download the next builds
+while you are currently evaluating one ([bug 999019]). Try this out, this
+will speed up the bisection process!
```

Comment:
Some grammar, some style feedback:

Change "This 0.35 release implement a new algorithm" -> "This release contains the implementation of a new algorithm". 

"Try this out, this will speed up the bisection process!" -> "On a fast enough connection, this can considerably speed up the bisection process by reducing the amount of time spent waiting for downloads to complete."